pkg/util/rlimit/rlimit_linux.go

```diff
@@ -22,6 +22,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-func RlimitNumFiles(maxOpenFiles uint64) {
+// NumFiles sets the linux rlimit for the maximum open files.
+func NumFiles(maxOpenFiles uint64) {
```

Comment:
I'm not sure about this rename, I guess this is a breaking change for users importing this package. Is there anything like this documented somewhere on how do deal with breaking changes or renames and such 🤔 ?

Reply:
I think the rename is fine... in my mind, we only need to worry about code in `staging` being imported by external applications. So as long as we update all uses in `k8s`, which I believe you did, we should be good to go.

What would you think of `SetNumFiles` as a function name? To me, `NumFiles` suggests is only accessing the value, not setting it.

Reply:
Good idea @mattjmcnaughton !  I've implemented this let me know what you think 🙇